### DISCUSSION TITLE
GitHub won't accept my letter nor my location ?

### DISCUSSION THREAD

**Discussion by lolouthefox** [community/community#137896] [GitHub Education]
### Hi everyone,

It seems like I can't get my GH education approved, are the informations about my school wrong or something? I do have a letter with the name of the school and a signature from school's head...

**Comment by AhmedFaizanDev** [comment]
Hey, maybe double-check your school info and try reaching out to GitHub support. That letter sounds good, but maybe something small is off.

**Comment by lolouthefox** [comment]
I've added my billing infos, but it's still not working, it still believes
I'm not on campus
